mongo_orchestration/servers.py

```diff
@@ -325,8 +326,10 @@ def start(self, timeout=300):
         if self.is_alive:
             return True
         try:
-            if self.cfg.get('dbpath', None) and self._is_locked:
+            dbpath = self.cfg.get('dbpath', None)
```

Comment:
None is the default return value for get().

Reply:
Done.